src/Uno.Extensions.Navigation.Toolkit/ServiceCollectionExtensions.cs

```diff
@@ -1,10 +1,39 @@
-﻿namespace Uno.Extensions;
+﻿using Uno.Extensions.Hosting;
+
+namespace Uno.Extensions;
 
 /// <summary>
 /// Extension methods for adding services to an <see cref="IServiceCollection" />.
 /// </summary>
 public static class ServiceCollectionExtensions
 {
+	private struct ToolkitViewInitializer : IRootViewInitializer
+	{
+		public ContentControl CreateDefaultView() => new ExtendedSplashScreen();
```

Comment:
Change this to create LoadingView, rather than SplashScreen - I'm thinking of the scenario where this gets used internally in the app where you're not likely to want a splash screen.